20220610.md

```diff
@@ -45,3 +45,124 @@ This last fortnight, over the 17 pull requests or activities reported in the ope
 
 * `docs.sailfishos.org`, website sources, **olf** is proposing [some formatting changes in the newly introduced release page](https://github.com/sailfishos/docs.sailfishos.org/pull/87), still under discussion with **rainemak**.
 * `osc`, the command line interface to work with OBS (the Open Build Service), **thaodan** and **mkosola** reached approval to [upgrade from 0.168.2 to 0.178.0](https://github.com/sailfishos/osc/pull/1).
+
+# Apps ahoy!
+
+In the early days of the Community News when things were only just getting started, we restricted the number of apps we covered to just four from those that received an update in the fortnight before. While trying out the different apps &mdash; and having the opportunity to really shake out the details &mdash; was a lot of fun, it was also disappointing to miss out all of the other wonderful apps that could have gone in.
+
+Writing an app is a labour of love, and that deserves recognition. And while the most important recognition comes from the app's users themselves (please, if you're using an app, take the opportunity to comment, rate or favourite it!), it's always gratifying to be able to give a small additional nod to the authors in this newsletter too.
+
+But while things have moved on, and we try now to be a bit more exhaustive with the apps that appear in this section, we still have to leave many deserving apps out. Among those are apps that are updated so frequently that we would have to include them every month if we were to stick to the principle. Apps like VideoWorks and Audioworks from prolific app-maintainer **poetaster**. And don't even get me started on all the apps receiving updates on OpenRepos and Chum, but which don't make it into harbour.
+
+So we hope you enjoy the app updates below, but while you read them, please remember that these still represent just a small portion of the Sailfish OS apps that are getting love and attention from their authors all the time.
+
+## Octava
+
+Octava from Rustem Abzalov (**arustg**) is a relatively new app, and yet in the last few months has seen multiple updates. It's not a complex app, and doesn't claim to be, billing itself as a "Simple Piano Keyboard". If you want to play simple melodies you might just get away with it, as long as your tune doesn't span multiple octaves, and doesn't contain chords. Octava prodtinkle the ivoriesuces a nice rich piano sound, with some nice animations to highlight the key being played. If you're itching to tinkle the digital ivories, this is your opportunity. The latest update provides some subtle user interface improvements. But let's hope **arustg** adds multitouch support so we can really bash out those choruses! Version 0.3.1 of Octava is available from both the Jolla Store and [OpenRepos](https://openrepos.net/content/arustg/frome).
```

Comment:
/prodtinkle the ivoriesuces/produces/ maybe ;)

Reply:
Thanks... I must have hit paste at the wrong time! Now fixed.